_docs_integrate/propose-attribute-to-peer.md

```diff
@@ -38,22 +38,23 @@ The Sender wants to propose an Attribute to the Recipient. To do this, the Sende
+| IdentityAttribute                                                                                       | Third party |     ✗     | `N/A`           | It makes no sense to propose an IdentityAttribute to the Recipient that is owned by a third party.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| RelationshipAttribute that exists in the context of a Relationship between the Sender and the Recipient | Sender      |     ✗     | `N/A`           | It makes no sense for the Sender to propose RelationshipAttributes to the Recipient that are owned by itself. Instead, it may be appropriate to proceed as described in the documentation for the [Create Attribute for peer]({% link _docs_integrate/create-attribute-for-peer.md %}) or [Read Attribute from peer]({% link _docs_integrate/read-attribute-from-peer.md %}) scenario and use the [CreateAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#createattributerequestitem) or the [ReadAttributeRequestItem]({% link _docs_integrate/data-model-overview.md %}#readattributerequestitem) accordingly. |
+| RelationshipAttribute that exists in the context of a Relationship between the Sender and the Recipient | Recipient   |     ✓     | `USER_DECISION` | With this combination, the Sender gives the Recipient the **one-time permission to create a RelationshipAttribute**, which is owned by the Recipient, whereby the Sender **proposes a value** for it as default option that might make sense.<br>Example: A company asks a new customer to subscribe to its newsletter and proposes the subscription as default once.                                                                                                                                                                                                                                                                   |
```

Comment:
"whereby" sounds a bit strange to me. How about "With this combination, the Sender gives the Recipient the **one-time permission to create a RelationshipAttribute**, which is owned by the Recipient, with the Sender **proposing a value** for it as default option that might make sense.